lms/djangoapps/django_comment_client/base/views.py

```diff
@@ -456,20 +480,36 @@ def delete_comment(request, course_id, comment_id):
     return JsonResponse(prepare_content(comment.to_dict(), course_key))
 
 
+def _vote_or_unvote(request, course_id, obj, value):
+    """
+    Vote or unvote for a thread or a response.
+    """
+    course_key = SlashSeparatedCourseKey.from_deprecated_string(course_id)
```

Comment:
@dianakhuang I didn't add this line – I just moved it here.  There are about 20 more places in the same file that use `SlashSeparatedCourseKey.from_deprecated_string()`, and not a single one using `CourseKey.from_string()`, so changing this one occurrence would feel weird.  Changing all of them seems outside the scope of this PR and would be better done in a separate PR.